gapic/src/main/com/google/gapid/perfetto/TraceView.java

```diff
@@ -72,7 +72,8 @@ public TraceView(Composite parent, Models models, Widgets widgets) {
 
   private static TraceComposite<State.ForSystemTrace> createTraceUi(
       Composite parent, Models models, Theme theme) {
-    return new TraceComposite<State.ForSystemTrace>(parent, models.analytics, models.perfetto, theme) {
+    return new TraceComposite<State.ForSystemTrace>(
+        parent, models.analytics, models.perfetto, theme, true) {
```

Comment:
Generally a fan of leaving a parameter comment like `/* fullView= */ true` for parameters that aren't as obvious

Reply:
Done.